tests/expired_service/common/actions.js

```diff
@@ -908,6 +908,36 @@ const actions = new Map([
       }
     ),
   ],
+  [
```

Comment:
No need to update this file as this is test data only (we should probably move it under a new `testdata/` folder, but definitely out of scope for this PR!).